crates/cxx-qt-gen/src/generator/rust/mod.rs

```diff
@@ -79,7 +79,11 @@ impl GeneratedRustBlocks {
 
 /// Generate the include line for this parsed block
 fn generate_include(parser: &Parser) -> Result<Item> {
-    let import_path = format!("cxx-qt-gen/{}.cxxqt.h", parser.cxx_file_stem);
+    let import_path = format!(
+        "{crate_name}/{}.cxxqt.h",
+        parser.cxx_file_stem,
+        crate_name = std::env::var("CARGO_PKG_NAME").unwrap_or_else(|_| "cxx-qt-gen".to_owned())
```

Comment:
is there a situation where `CARGO_PKG_NAME` is actually empty? wondering if we should error out instead of silently falling back to `cxx-qt-gen` ? As what happens if two crates try to write to that same path ?

Reply:
Refactored so that cxx-qt-build now writes the header prefix to disk and we can then read it back when building.

It's then still empty when testing so we still fall back to cxx-qt-gen and don't need to adjust our tests :)